drivers/adc/adc_ads1119.c

```diff
@@ -21,11 +21,11 @@
 LOG_MODULE_REGISTER(ADS1119, CONFIG_ADC_LOG_LEVEL);
 
 
-#define ADS1119_CONFIG_VREF(x) ((x) & BIT(0))
-#define ADS1119_CONFIG_CM(x) ((x) & BIT(1))
-#define ADS1119_CONFIG_DR(x) ((x) & (BIT_MASK(2) << 2))
-#define ADS1119_CONFIG_GAIN(x) ((x) & BIT(4))
-#define ADS1119_CONFIG_MUX(x) ((x) & (BIT_MASK(3) << 5))
+#define ADS1119_CONFIG_VREF(x) (FIELD_PREP(GENMASK(0, 0), x))
+#define ADS1119_CONFIG_CM(x) (FIELD_PREP(GENMASK(1, 1), x))
+#define ADS1119_CONFIG_DR(x) (FIELD_PREP(GENMASK(3, 2), x))
+#define ADS1119_CONFIG_GAIN(x) (FIELD_PREP(GENMASK(4, 4), x))
```

Comment:
sorry to be unclear, didn't mean to convert these BIT to GENMASK, the BIT and BIT_MASK made more sense, but keep the FIELD_PREP

Reply:
Changed in [e0acd25](https://github.com/zephyrproject-rtos/zephyr/pull/70120/commits/e0acd2527e715df7dd20c9de4e772a64a21a6b0f).